src/Pack/CmdLn/Types.idr

```diff
@@ -24,7 +24,7 @@ Arg PkgQuery where
   argDesc_ = "[mode] <query>"
 
   readArg ("dep" :: s :: t)     = Just (MkQ Dependency s, t)
-  readArg ("modules" :: s :: t) = Just (MkQ Module s, t)
+  readArg ("module" :: s :: t) = Just (MkQ Module s, t)
```

Comment:
About code style: I think it's good to preserve already present alignment of `=`-signs